apps/prairielearn/src/pages/instructorStudents/instructorStudents.html.tsx

```diff
@@ -194,9 +194,9 @@ function StudentsCard({
                 aria-label="Search by UID, name or email."
                 placeholder="Search by UID, name, email..."
                 value={globalFilter}
-                onInput={(e) => {
+                onInput={async (e) => {
                   if (!(e.target instanceof HTMLInputElement)) return;
-                  setGlobalFilter(e.target.value);
+                  await setGlobalFilter(e.target.value);
```

Comment:
I'd think that a `void` annotation would be better here.

Reply:
Done in ed32c0fd05e656102949ea224dc4336fbd3fba0f.